src/artwork.c

```diff
@@ -392,6 +393,64 @@ static struct online_source musicbrainz_source =
   };
 
 
+/* -------------------------------- HELPERS -------------------------------- */
+
+#define TMP_BUF_SIZE 1024
+static int artwork_read_byurl(struct evbuffer *evbuf, const char *url);
```

Comment:
Avoid the forward by moving artwork_read_json_byurl down

Reply:
artwork_read_byurl() and artwork_read_json_byurl() call each other, so one needs forward decleration.